Framework/Algorithms/inc/MantidAlgorithms/BinaryOperation.h

```diff
@@ -111,8 +111,8 @@ class DLLExport BinaryOperation : public API::Algorithm {
   /// masked. If so then the output spectra has zeroed data
```

Comment:
As a side note. I notice that this header is passing shared_pointers by value where it probably shouldn't be. Not sure it's worth you fixing that here though.